ckan-client-guide/README.md

```diff
@@ -4,113 +4,523 @@ Guide to interacting with [CKAN](/ckan) for power users such as data scientists,
 
 This guide is about adding and managing data in CKAN programmatically and it assumes:
 
-* You are familiar with key concepts like metadata, data etc.
-* You are working programmatically e.g. in one of Python, R, JavaScript, etc.
+* You are familiar with key concepts like metadata, data, etc.
+* You are working programmatically with a programming language such as Python, JavaScript or R (_coming soon_).
 
 ## Frictionless Formats
 
-Clients use [Frictionless formats][f11s] by default for describing dataset and resource objects passed to client methods. Internally, we then use the a CKAN<=>Frictionless Mapper ([js][c2f], [python][c2fpy] to convert to CKAN formats before calling the API. Thus, you can use Frictionless Formats by default with the client. (As CKAN moves to Frictionless to default this will gradually become unnecessary).
+Clients use [Frictionless formats](https://specs.frictionlessdata.io/) by default for describing dataset and resource objects passed to client methods. Internally, we then use the a *CKAN <=> Frictionless Mapper* (both [in JavaScript]( https://github.com/datopian/frictionless-ckan-mapper-js ) and [in Python](https://github.com/frictionlessdata/frictionless-ckan-mapper)) to convert objects to CKAN formats before calling the API. **Thus, you can use _Frictionless Formats_ by default with the client**.
 
-[f11s]: https://specs.frictionlessdata.io/
-[c2f]: https://github.com/datopian/frictionless-ckan-mapper-js
+::::tip
+
+As CKAN moves to Frictionless to default this will gradually become unnecessary.
+
+::::
 
 ## Quick start
 
-This guide currently uses a Python idiom by default (e.g. lower case, underscore). You can adapt to R and Javascript as needed and we provide alternative examples where possible.
+Most of this guide has Python programming language in mind, including its [convention regading using _snake case_ for instances and methods names](https://www.python.org/dev/peps/pep-0008/#descriptive-naming-styles).
+
+If needed, you can adapt the instructions to JavaScript and R (coming soon) by using _camel case_ instead — for example, if in the Python code we have  `client.push_blob(…)`, in JavaScript it would be `client.pushBlob(…)`.
 
 ### Prerequisites
 
-Install the SDK for your language and configure it.
+Install the client for your language of choice:
 
-* Python: https://github.com/datopian/ckan3-py-sdk#install) 
-* Javascript: ...
-* R: coming soon
+* For Python, [use `pip install ckan-client`](https://github.com/datopian/ckan-client-py#install)
+* For JavaScript, [use `npm install https://github.com/datopian/ckan-client-js.git`](https://github.com/datopian/ckan-client-js#install)
```

Comment:
You don't need the npm install instructions - just link to install instructions (that way we avoid things getting repeated).